From df4d74303bb7a97da50a587eba032b0e0a35ef9b Mon Sep 17 00:00:00 2001
From: "neil.shi" <neil.shi@quectel.com>
Date: Thu, 1 Jun 2023 13:45:51 +0800
Subject: [PATCH 046/188] wifi: ath11k: add log for
 HE_MODE_SU_TX_BFER/HE_MODE_MU_TX_BFEE

Signed-off-by: neil.shi <neil.shi@quectel.com>
Cherry-picked-from: 1970502fc143 ("Subject: [PATCH 40/42] wifi: ath11k: add log for HE_MODE_SU_TX_BFER/HE_MODE_MU_TX_BFEE")
Signed-off-by: Eugen Hristev <eugen.hristev@collabora.com>
Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
---
 drivers/net/wireless/ath/ath11k/mac.c | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/drivers/net/wireless/ath/ath11k/mac.c b/drivers/net/wireless/ath/ath11k/mac.c
index 6133d206a..13fdeda29 100644
--- a/drivers/net/wireless/ath/ath11k/mac.c
+++ b/drivers/net/wireless/ath/ath11k/mac.c
@@ -3085,6 +3085,27 @@ static bool ath11k_mac_vif_recalc_sta_he_txbf(struct ath11k *ar,
 			hemode |= FIELD_PREP(HE_MODE_SU_TX_BFER, HE_SU_BFER_ENABLE);
 	}
 
+	ath11k_info(ar->ab, "mac0-5 cap %x-%x-%x-%x-%x-%x\n",
+		he_cap_elem.mac_cap_info[0],
+		he_cap_elem.mac_cap_info[1],
+		he_cap_elem.mac_cap_info[2],
+		he_cap_elem.mac_cap_info[3],
+		he_cap_elem.mac_cap_info[4],
+		he_cap_elem.mac_cap_info[5]);
+	ath11k_info(ar->ab, "phy0-5 cap %x-%x-%x-%x-%x-%x\n",
+		he_cap_elem.phy_cap_info[0],
+		he_cap_elem.phy_cap_info[1],
+		he_cap_elem.phy_cap_info[2],
+		he_cap_elem.phy_cap_info[3],
+		he_cap_elem.phy_cap_info[4],
+		he_cap_elem.phy_cap_info[5]);
+	ath11k_info(ar->ab, "phy6-10 cap %x-%x-%x-%x-%x\n",
+		he_cap_elem.phy_cap_info[6],
+		he_cap_elem.phy_cap_info[7],
+		he_cap_elem.phy_cap_info[8],
+		he_cap_elem.phy_cap_info[9],
+		he_cap_elem.phy_cap_info[10]);
+	ath11k_info(ar->ab, "WMI_VDEV_PARAM_SET_HEMU_MODE 3 0x%x\n", hemode);
 	ret = ath11k_wmi_vdev_set_param_cmd(ar, arvif->vdev_id, param, hemode);
 	if (ret) {
 		ath11k_warn(ar->ab, "failed to submit vdev param txbf 0x%x: %d\n",
-- 
2.55.0

